ecosystem/typescript/sdk/src/transaction_builder/builder.ts

```diff
@@ -261,6 +245,35 @@ export class TransactionBuilderABI {
       payload = new TransactionPayloadScript(new Script(funcABI.code, typeTags, scriptArgs));
     }
 
+    return payload;
+  }
+
+  /**
+   * Builds a RawTransaction
+   * @param func Fully qualified func names, e.g. 0x1::Coin::transfer
+   * @param ty_tags TypeTag strings.
+   * @example Below are valid value examples
+   * ```
+   * // Structs are in format `AccountAddress::ModuleName::StructName`
+   * 0x1::aptos_coin::AptosCoin
+   * // Vectors are in format `vector<other_tag_string>`
+   * vector<0x1::aptos_coin::AptosCoin>
+   * bool
+   * u8
+   * u64
+   * u128
+   * address
+   * ```
+   * @param args Function arguments
+   * @returns RawTransaction
+   */
+  build(func: string, ty_tags: string[], args: any[]): RawTransaction {
```

Comment:
with these changes, dApp will use the `buildTransactionPayload()` to build a payload func to then send to the wallet and the wallet will use this `build()` to build a RawTransaction?

Reply:
I believe so, this way the dapp doesn't have to worry about `chainId`, `sequenceNumber` and other parameters potentially configurable within the wallet